src/main/java/io/supertokens/session/accessToken/AccessToken.java

```diff
@@ -322,22 +322,32 @@ public static VERSION getAccessTokenVersionForCDI(SemVer version) {
     }
 
     public static VERSION getVersionFromString(String versionString) {
-        switch (versionString) {
-            case "1":
-                return VERSION.V1;
-            case "2":
-                return VERSION.V2;
-            case "3":
-                return VERSION.V3;
-            case "4":
-                return VERSION.V4;
-            default:
-                throw new IllegalArgumentException("Invalid version string: " + versionString);
-        }
+        return VERSION.valueOf("V" + versionString);
     }
 
     public static class AccessTokenInfo {
-        public static String[] protectedPropNames = {
+
+        static String[] protectedPropNamesV2 = {
```

Comment:
remove these dupluicate arrays and rename requiredPropsV2 to requiredAndProtectedPropsV2

Reply:
Done